.github/workflows/update-sandbox.yml

```diff
@@ -0,0 +1,77 @@
+name: sandbox-release
+on:
+  release:
+    types: [published]
```

Comment:
Should we also run this on draft release?
Afair, we discussed drafting release to push it to sandbox for some time.